packages/inter-protocol/test/price/snapshots/test-fluxAggregatorKit.js.md

```diff
@@ -30,14 +30,24 @@ Generated by [AVA](https://avajs.dev).
           timer: {
             iface: 'Alleged: ManualTimer',
           },
-          timestamp: 6n,
+          timestamp: {
```

Comment:
ask @dckc about whether CLI tools pay attention to this format and need to change

Reply:
I can't think of any, but I'm juggling too much to be confident about my answer.

@turadg ?
@ivanlei ? do we have partners relying on this part of our RPC API?

Reply:
I don't think they're checking Agoric chain time because they just push the latest. I'll send this to them to correct if necessary.

Reply:
Corrected by @jacquesvcritien . I was thinking about whether they need to be concerned with the brand but this shape change will affect their code. They depend on `startedAt` for:
- To know when the last round was started in order to start a new round after 10 minutes (startedAt)
- To predict time left for next round for monitoring.

I've recommended using `absValue()` to access the time, which reads the bigint from branded and unbranded.

Reply:
ok, when this lands, the external format will change, thanks for giving them the heads up